jekyll/faq.md

```diff
@@ -10,15 +10,26 @@ css_class: faq
 
 ## Why yet another programming language?
 
-Nim is one of the very few _programmable_ statically typed languages, and one
-of the even fewer that produces native binaries that require no runtime or
-interpreter.
+Nim is one of the very few _programmable_ statically typed languages, and
+attempts to combine the speed and memory efficency of C, an expressive syntax,
+memory safety and multiple target languages.
 
+## How fast is Nim?
 
-## What have been the major influences in the language's design?
+Benchmarks show it to be comparable to C. Some language features (methods,
+closures, message passing) are not yet as optimized as they could and will be.
+The only overhead Nim has over C is the GC which has been tuned
+for years but still needs some work.
```

Comment:
Does it still need work? I think it's better to focus on positives here anyway.

Reply:
"which has been tuned for years and is competitive with other GC implementations. It can also be easily avoided since a heap in Nim is attached to a thread."